backend/src/models/consumptions.ts

```diff
@@ -32,10 +32,9 @@ export const getFamilyDependentBalanceProduct = async (family: Family): Promise<
   //Filter benefit by family date
   const familyBenefitsFilterDate = familyBenefits
     .filter((benefit) => {
-      const isAfter = moment(benefit.date).isSameOrAfter(moment(family.createdAt || moment()));
-      let isBefore = true;
-      if (family.deactivatedAt) isBefore = moment(benefit.date).isBefore(moment(family.deactivatedAt));
-      return isAfter && isBefore ? benefit : null;
+      const isSameMonthYear = moment(benefit.date).isSame(moment(family.createdAt || moment()), 'month');
```

Comment:
Não vai funcionar lidar com o mês assim. Quando virar o mês a família vai perder o benefício porque não está mais no mesmo mês.
O ideal é conferir se o createdAt está depois do início do mês e se a data atual está depois do `date` do mês.